packages/blade/src/components/Icons/_Svg/Path/Path.d.ts

```diff
@@ -1,6 +1,8 @@
-export type NonDashedProps<T> = {
-  [K in keyof T]: K extends `${string}-${string}` ? never : T[K];
-};
+// https://github.com/microsoft/TypeScript/issues/12936#issuecomment-368244671
```

Comment:
Can add some more explanation inline here perhaps, hard to understand the linked comment 😅

Reply:
Will try. Kinda hard to explain that huge discussion in a few lines though.

Reply:
Just a description on what Exact does. Don't need to add all the context on why we need and all

Reply:
Added 

<img width="869" alt="image" src="https://user-images.githubusercontent.com/35374649/187179510-56e43fe1-26c9-4a84-b4d4-c62ed88aed5e.png">